sa/sa.go

```diff
@@ -1631,8 +1814,16 @@ func (ssa *SQLStorageAuthority) GetOrderForNames(
 		return nil, err
 	}
 
-	// Get & return the order
-	return ssa.GetOrder(ctx, &sapb.OrderRequest{Id: &orderID})
+	// Get the order
+	order, err := ssa.GetOrder(ctx, &sapb.OrderRequest{Id: &orderID})
+	if err != nil {
+		return nil, err
+	}
+	// Only return a pending order
```

Comment:
This function's documentation needs updating to reflect the "only return pending" behavior.

Reply:
Fixed in 7cb8c00